### ISSUE TITLE
Kuh wird schneller

### ISSUE THREAD

**Issue by whatatoddler** [whatatoddler/COWLIFE#16]
Habe gerade festgestellt, dass die Kuh jedesmal schneller wird, wenn man wiederholt auf Play drückt. Ist das ein Bug oder ein Feature? :D

**Comment by MalibuKanada** [comment]
Right, ist sowas wie ein Bug. Bei Klick auf Start bekommt das Canvas die Anweisung die Kuh auf Klick laufen zu lassen. Wenn erneut geklickt wird gibts nen Doppelevent, wäre am sinnvollsten, wenn der Start Button gleich ins Canvas kommt, also erstmal nicht mehrfach klicken :)

**Comment by whatatoddler** [comment]
NEUE IDEE: Das Spiel beschreibt den Werdegang des Spitzensportlers KUHsain Bolt! Haha!
